Sources/Datadog/RUMMonitor.swift

```diff
@@ -302,12 +304,13 @@ public class RUMMonitor {
     /// - Parameters:
     ///   - type: the User Action type
     ///   - attributes: custom attributes to attach to the User Action.
-    public func registerUserAction(type: RUMUserActionType, attributes: [AttributeKey: AttributeValue]? = nil) {
+    public func registerUserAction(type: RUMUserActionType, name: String, attributes: [AttributeKey: AttributeValue]? = nil) {
```

Comment:
It needs the `public` API comment update with explaining to the user what it is.